README.md

```diff
@@ -33,23 +33,158 @@ Main features
 - :microscope: Tiny and tree-shakeable
 - :muscle: Supports TypeScript
 
-## Docs
+## Table of Contents
+
+- [Installation](#installation)
+- [API](#api)
+  - [connectObservable](#connectObservable)
+  - [connectFactoryObservable](#connectFactoryObservable)
+  - [distinctShareReplay](#distinctShareReplay)
+  - [SUSPENSE](#SUSPENSE)
+  - [suspend](#suspend)
+  - [suspended](#suspended)
+  - [switchMapSuspended](#switchMapSuspended)
+  - [createInput](#createInput)
+- [Examples](#examples)
+
+
+## Installation
+    npm install re-rxjs
+
+## API
+
+### connectObservable
+```ts
+const [useCounter, sharedCounter$] = connectObservable(
+  clicks$.pipe(
+    scan(prev => prev + 1, 0),
+    startWith(0),
+  )
+)
+```
+Returns a hook that provides the latest update of the observable, and an enhanced 
```

Comment:
Returns a hook that provides the latest update of the **accepted** observable, and an enhanced